cloud/aws/modules/dbaccess/ip.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+read -r input
+region=$(echo $input | grep -o '"region": *"[^"]*"' | sed 's/"region": *"\([^"]*\)"/\1/')
```

Comment:
(suggestion) Add some comments on what region is expected to be. It says ubuntu so I don't want to assume is the AWS region. Maybe even validate it's an allowed value if it's a fixed enum.